internal/pkg/template/env.go

```diff
@@ -41,14 +39,38 @@ type EnvOpts struct {
 	ArtifactBucketARN         string
 	ArtifactBucketKeyARN      string
 
-	ImportVPC      *config.ImportVPC
-	VPCConfig      *config.AdjustVPC
+	VPCConfig      VPCConfig
 	ImportCertARNs []string
-	Telemetry      *config.Telemetry
+	Telemetry      *Telemetry
 
 	LatestVersion string
 }
 
+type VPCConfig struct {
+	Imported *ImportVPC
+	Managed  ManagedVPC
```

Comment:
nit: can we add a comment here
```suggestion
	Imported *ImportVPC // If not-nil, use the imported VPC settings instead of the defaults from ManagedVPC.
	Managed  ManagedVPC
```

Reply:
I tweaked the wording a little:
```
// If not-nil, use the imported VPC resources instead of the Managed VPC.
```
Because the managed VPC could be configured by user input (e.g. configuring cidr, az)